google-api-go-generator/gen.go

```diff
@@ -2224,6 +2224,10 @@ func (meth *Method) generateCode() {
 		contentType = `"application/json"`
 	}
 	pn(`reqHeaders := gensupport.SetHeaders(c.s.userAgent(), %s, c.header_ )`, contentType)
+	apiVersion := meth.m.APIVersion
```

Comment:
nit: Instead of inlining this to every function, could we add a new param to SetHeaders and send this in there?